### PR TITLE
Option to pass release flag

### PR DESCRIPTION
**Pull request by dzimine**
Make it easy to use this Vagrant to get the dev trunk version. 

Updated scripts and adjusted README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,16 +46,27 @@ The WebUI is available at https://192.168.16.20. The default st2admin user crede
...
+To master StackStorm and understand how things are wired together, we strongly encourage you to
+[eventually] install StackStorm manually, following installation instructions. You can still
+benefit from this Vagrantfile to get the Linux VM up and running: follow instructions to
+install Vagrant & VirtualBox to get a Linux VM, and simply comment out the
+`st2.vm.provision "shell"...` section in your `Vagrantfile` before running `vagrant up`.
 
-If you want to install StackStorm manually, follow the
-instructions to install Vagrant & VirtualBox to get a Linux VM, simply comment out the
-`st2.vm.provision "shell"...` section in your `Vagrantfile`.
+
```
Comment (humblearner) [inline README.md]: Can we add the curl command here or point the users to the docs to customize/fine tune their installation ? Also, move this after Customize your st2 installation section.

File: README.md [hunk 2]
```diff
@@ -46,16 +46,27 @@ The WebUI is available at https://192.168.16.20. The default st2admin user crede
...
+## Customize your st2 installation
```
Comment (humblearner) [inline README.md]: Rename this to `Customize Vagrantfile for simple installation` or something similar.

File: scripts/install_st2.sh [hunk 3]
```diff
@@ -1,19 +1,30 @@
 #!/bin/bash
 
+# Select between recent stable (e.g. 1.4) or recent unstable (e.g. 1.5dev)
+if [[ $# > 2 ]]; then
+  if [[ $3 == "stable" ]] || [[ $3 == "unstable" ]]
+  then
+    RELEASE_FLAG="--$3"
+  else
+    echo -e "Use 'stable' for recent stable release, or 'unstalbe' to live on the edge."
```
Comment (lakshmi-kannan) [inline scripts/install_st2.sh]: typo in unstable

File: README.md [hunk 4]
```diff
@@ -127,7 +127,7 @@ To use this option, uncomment the line and change the location of `"path/to/fold
 existing directory on your laptop.
 
 By the time you read this hint, your VM is most likely already up and running. Not to worry: just
-uncomment the above mentioned line in your `Vagrantfile` and run `vagrant reload`. This will restart
+uncomment the above mentioned line in your `Vagrantfile` and run `vagrant reload --no-provision`. This will restart
```
Comment (dzimine) [inline README.md]: Why do we need --no-provision, it is there by default. I tested it again. Do you see different behaviour?

File: README.md (hunk 1 above)
Comment (dzimine) [inline README.md]: Moved, but prefer to reference to docs.

**Comment by lakshmi-kannan** [comment]
LGTM.

**Pull request merged**